plugins/modules/postgresql_ext.py

```diff
@@ -161,12 +161,23 @@
 '''
 
 RETURN = r'''
-query:
+queries:
```

Comment:
I think it is not a major change, buy a simple documetation fix, so can be retained in this PR. In final exit_json of ext module the returned parameter name was 'queries' also before my PR (was 'query' only in module documentation, maybe for a missing update in a past PR).

Reply:
i think it's a major change because someone can rely on this ret val in their playbooks (similarly to like we do in our tests when registering `result` and then check what values it has in code), so it's a breaking interface change

Reply:
Looking postgresql_ext module code at current master branch (before my PR).

Its documentation report a return value named 'query':

https://github.com/ansible-collections/community.postgresql/blob/6a5ddbf13c9a120da0b1142e044519bd2ecf10d0/plugins/modules/postgresql_ext.py#L163_L170

But looking in the code, the only exit_json of the module include a return value named 'queries':

https://github.com/ansible-collections/community.postgresql/blob/6a5ddbf13c9a120da0b1142e044519bd2ecf10d0/plugins/modules/postgresql_ext.py#L471

This is the situation before my PR and there is a clear conflict between the documentation (first link above) and the code (second link above).

Now the documentation cannot be a reference for playbooks; on the contrary, if a collection user see the documentation he think that the module postgresql_ext returned a parameter named 'query' but instead he found after a test the returned parameter Is named 'queries'.

My PR fix this conflict, replacing the wrong name 'query' with the correct name 'queries' present in the code. It is a clear documentation bugfix to help developers. In PR exit_json is changed only to insert new returned parameters, without touching 'queries'.

Reply:
oh, sorry, right, sorry for the noise, marking my comment as resolved:)